### ISSUE TITLE
Best clipboard manager ever, however I need help

### ISSUE THREAD

**Issue by GmanAtheist** [sabrogden/Ditto#743]
I have been using Ditto for years in win 7, 8, 10 and now 11. Just one small issue. When I copy a screen area , a preview appear above taskbar right side. That sometimes mess up my screen recordings. I can not find a setting in Ditto to turn off that preview image appearing. Please help.

**Comment by sabrogden** [comment]
I don't think that is Ditto.  Pressing shift- windows - s then selecting some area of the screen to copy will cause Windows Snipping Tool to show just above the task bar, clicking it opens the copied area in Snipping Tool.

I'm not seeing a way to not pop this up in the snipping tool settings.

**Comment by effctcodes** [comment]
When i screenshot with shift-win-s in Windows 10, i get the notification from app “Snip & Sketch”.  Blocking this app’s notifications worked:
![image](https://github.com/user-attachments/assets/331fd57e-79d3-45f3-adaf-eca26ea70921)
You can turn them back on by pressing win-a then clicking _Manage notifications_ in the top right corner:
![image](https://github.com/user-attachments/assets/e7e561af-5669-4789-9d2b-dae7782025f1)

**Comment by Chealer** [comment]
@GmanAtheist wrote:
> So I presume there is no support for this app. :/

This is not a support forum. Are you reporting an issue here? In any case, you should indicate the version you use.
